### PR TITLE
Supplement processing options

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing/configuration.rst [hunk 1]
```diff
@@ -16,7 +16,9 @@ structured in separate blocks that you can select on the left-hand side of the
 dialog.
 
 Along with the aforementioned :guilabel:`Output folder` entry, the
```
Comment (DelazJ) [inline source/docs/user_manual/processing/configuration.rst]: Afaics, there's no previous "output folder" mention and with the current dialog structure, this block belongs to the end of the section, before the crs warning

File: source/docs/user_manual/processing/configuration.rst [hunk 2]
```diff
@@ -40,6 +42,19 @@ Select the style file (:file:`.qml`) that you want for each output and press
...
+  again with different parameters, or to better check the output that is written
+  to the log tab), check this option
+* :guilabel:`Pre-execution script` and :guilabel:`Post-execution script file`.
+  These parameters refer to scripts written using the processing scripting
+  functionality, and are explained in the section covering scripting and the
+  console.
+* :guilabel:`Show algorithms with known issues`
+* :guilabel:`Show layer CRS definition in selection boxes`
+* :guilabel:`Show tooltip when there are disabled providers`
 * :guilabel:`Use filename as layer name`. The name of each resulting layer created
```
Comment (DelazJ) [inline source/docs/user_manual/processing/configuration.rst]:
```suggestion
* :guilabel:`Prefer output filename to layer names`. The name of each resulting layer created
```
hoping that the behavior is the same

File: source/docs/user_manual/processing/configuration.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/processing/configuration.rst]: The list of options was very much incomplete.
I have now listed all the entries I found in 3.10, but have not provided many explanations.
I think that could be done in later PR.

File: source/docs/user_manual/processing/configuration.rst (hunk 2 above)
Comment (havatv) [reply source/docs/user_manual/processing/configuration.rst]: Me too. Done in the latest commit.

**Pull request merged**